deployment/src/main/java/io/quarkus/tika/deployment/TikaProcessor.java

```diff
@@ -90,6 +92,16 @@ public void registerTikaCoreResources(BuildProducer<NativeImageResourceBuildItem
         resource.produce(new NativeImageResourceBuildItem("org/apache/tika/parser/external/tika-external-parsers.xml"));
     }
 
+    @BuildStep
+    void registerForReflection(BuildProducer<ReflectiveClassBuildItem> reflectiveClass) {
+
+        final List<String> classNames = new ArrayList<>();
```

Comment:
Isn't easier to pass these classes directly in the `builder(...)` method?

Reply:
Doh!  Yes I will fix tomorrow.